.github/workflows/publish-docker.yaml

```diff
@@ -52,5 +52,9 @@ jobs:
           registry: ${{ env.HUB }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up QEMU
```

Comment:
QEMU, interesting. Anything needs to be added for release document when do container image? I feel there are some extra setups.

Reply:
> QEMU, interesting. Anything needs to be added for release document when do container image? I feel there are some extra setups.

<img width="754" alt="Screen Shot 2021-11-17 at 22 37 14" src="https://user-images.githubusercontent.com/15965696/142220847-71c032eb-5359-4757-9c8e-50f964f5fe01.png">

For maintainers using Mac OS, a Docker Desktop is just enough to release the images, as tested in https://hub.docker.com/repository/docker/kezhenxu94/oap/tags?page=1&ordering=last_updated

Reply:
How about Linux(Ubuntu for example)? I was building all release(binary, source tar and image) on GCP Linux